packages/run-protocol/src/proposals/econ-behaviors.js

```diff
@@ -261,6 +265,9 @@ export const setupReserve = async ({
     },
   },
 }) => {
+  trace('setupReserve');
+  await reserveInstallation;
```

Comment:
what's this for? why await it and then throw away the resolution?

Reply:
earlier feedback on missing producers. I'll do some cleanup since they're confusing